src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -54,6 +54,16 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
         daemon_spec.extra_files = {'ceph-nvmeof.conf': gw_conf}
         daemon_spec.final_config, daemon_spec.deps = self.generate_config(daemon_spec)
         daemon_spec.deps = []
+        # Notify monitor about this gateway creation
+        cmd = {
+            'prefix': 'nvme-gw create',
+            'id': name,
+            'group': spec.group,
+            'pool': spec.pool
+        }
+        _, _, err = self.mgr.mon_command(cmd)
+        if err:
+            self.mgr.log.error(f"Unable to send monitor command {cmd}, error {err}")
```

Comment:
If the monitor command fails but cephadm places or removes an nvmeof gateway on a node will the system automatically recover? Are there any UX implications: like if this fails what would an admin have to do to fix/recover things? If it isn't "self healing" we may need to something more than just a log line that the admin may not know to look for.

Reply:
Good point, this should be a fatal error, failing the daemon creation, see [diff](https://github.com/ceph/ceph/pull/54671/commits/a7d845b7ee9f84addaf3ce248f806f5cf2fdd2e6)